skyportal/models.py

```diff
@@ -1866,6 +1915,20 @@ class Spectrum(Base):
         sa.String, doc="Original file name that was passed to upload the spectrum."
     )
 
+    owner_id = sa.Column(
+        sa.ForeignKey('users.id', ondelete='CASCADE'),
+        nullable=True,
```

Comment:
same

Reply:
Fair enough

Reply:
This will make the migration a bit harder though  - what should we use for data that are already in there ?

Reply:
id=3

Reply:
I think no one else has uploaded any other data there yet